### PR TITLE
feat: pre-fill feedback form with details from error dialog

### PR DESCRIPTION
**Pull request by ujjwal-cyph**
Fixes https://app.clickup.com/t/9002019994/PRF-6442
Fixes https://app.clickup.com/t/9002019994/PRF-6488

### CODE REVIEW

**Comment by rohanagarwal94** [comment]
Task linked: [PRF-6442 Link pre-filled Support page to Help in error dialog](https://app.clickup.com/t/9002019994/PRF-6442)

**Comment by rohanagarwal94** [comment]
Task linked: [PRF-6442 Link pre-filled Support page to Help in error dialog](https://app.clickup.com/t/9002019994/PRF-6442)

**Comment by rohanagarwal94** [comment]
Task linked: [PRF-6442 Link pre-filled Support page to Help in error dialog](https://app.clickup.com/t/9002019994/PRF-6442)

**Comment by rohanagarwal94** [comment]
Task linked: [PRF-6442 Link pre-filled Support page to Help in error dialog](https://app.clickup.com/t/9002019994/PRF-6442)

**Comment by rohanagarwal94** [comment]
Task linked: [PRF-6442 Link pre-filled Support page to Help in error dialog](https://app.clickup.com/t/9002019994/PRF-6442)

**Comment by rohanagarwal94** [comment]
Task linked: [PRF-6442 Link pre-filled Support page to Help in error dialog](https://app.clickup.com/t/9002019994/PRF-6442)

File: packages/cysync-core/src/dialogs/ContactSupport/context/index.tsx [hunk 1]
```diff
@@ -72,13 +72,19 @@ export const ContactSupportDialogContext: Context<ContactSupportDialogContextInt
     {} as ContactSupportDialogContextInterface,
   );
 
-export interface ContactSupportDialogProviderProps {
+export interface IContactSupportDialogProps {
+  providedDescription?: string;
+  errorCategory?: string;
```
Comment (irshadCypherock) [inline packages/cysync-core/src/dialogs/ContactSupport/context/index.tsx]: Create a type for this

**Review by irshadCypherock — CHANGES_REQUESTED** [review]

File: packages/cysync-core/src/dialogs/ContactSupport/context/index.tsx (hunk 1 above)
Comment (ujjwal-cyph) [reply packages/cysync-core/src/dialogs/ContactSupport/context/index.tsx]: added

**Review by irshadCypherock — APPROVED** [review]

**Pull request merged**